cmd/chihaya/main.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/chihaya/chihaya/server"
 	"github.com/chihaya/chihaya/tracker"
 
+	_ "github.com/chihaya/chihaya/middlewares/ip"
```

Comment:
It kinda makes sense - then again it would be in a subpackage of `github.com/chihaya/chihaya/server`, but it's definitely no server. (One might even go so far as to say that `github.com/chihaya/chihaya/server/store/middlewares` could be an implementation of `github.com/chihaya/chihaya/server/store`, just like `memory` is)

I think I'll move it, just to keep `chihaya/middlewares` nice and clean for non-server-dependent middlewares.